### PR TITLE
Local wizards

### PR DESCRIPTION
**Pull request by benlangfeld**
This removes legacy addon support and permits installing addons / wizards from a local path relative to the source directory:

```yaml
installer: true
wizards:
  - crohr/addon-mysql
  - ./packaging/wizards/myapp
```

See https://github.com/crohr/pkgr/issues/54

### CODE REVIEW

**Comment by benlangfeld** [comment]
That failure was a timeout and shouldn't repeat itself. @crohr, maybe you could restart the build?

**Comment by benlangfeld** [comment]
This is not yet good to go, and is lacking test coverage showing why. I'm adding that in.

**Comment by benlangfeld** [comment]
I think this is ready for review, @crohr.

File: spec/fixtures/.release [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-addons:
```
Comment (crohr) [inline spec/fixtures/.release]: This is actually a valid `.release` file, as can be written by the ruby buildpack. Is there a reason why you removed it?

**Comment by crohr** [comment]
Looks very fine to me, thanks! If you could just reply to my comment on the `.release` file, that would be great.

File: spec/fixtures/.release (hunk 1 above)
Comment (benlangfeld) [inline spec/fixtures/.release]: The reason would be ignorance. Fixed in a follow-up commit and then squashed.

**Comment by benlangfeld** [comment]
Complaints fixed. This should be good to go. How long might it take for this to reach packager.io, @crohr?

**Comment by crohr** [comment]
@benlangfeld currently running integration tests, so if everything goes well it should be live tomorrow. I'll let you know once it's available.

**Comment by benlangfeld** [comment]
Excellent, thanks. Out of curiosity, are those integration tests part of the hosted service, or something that can be used against open-source pkgr?

**Comment by benlangfeld** [comment]
Looks like this is now live on packager.io, and a build completed successfully with it: https://packager.io/gh/att-innovate/SIPtreadmill/build_runs/60. Thanks!

**Comment by crohr** [comment]
Yes it is now live on [packager.io](https://packager.io). Thanks for letting me know that it works for you!

**Pull request merged**